R/data.R

```diff
@@ -56,8 +56,8 @@ NULL
 #'
 #' @docType data
 #'  
-#' @description The data-set contains a matrix of about 600
-#' individuals and 1,000 variables. These variables are real-world
+#' @description The data-set contains a matrix of about 574
```

Comment:
Thanks @jdblischak while you are on it, could you change the `about 574` to just `574`? :)

Reply:
No problem. And what about the vignettes? They say approx 600? Do you want to keep that or switch to the exact number?

Reply:
@gaow I updated `R/data.R` to be only `574`